Peekabook/Peekabook/Presentation/UserSearch/VC/UserSearchVC.swift

```diff
@@ -32,12 +75,80 @@ final class UserSearchVC: UIViewController {
 extension UserSearchVC {
     
     private func setUI() {
-        self.view.backgroundColor = .peekaBeige
+        self.view.backgroundColor = UIColor.peekaBeige
     }
     
     private func setLayout() {
+        view.addSubviews([searchBarContainerView,
+                          userSearchTableView,
+                          headerView])
+        headerView.addSubviews([backButton, searchTitleLabel, headerUnderlineView])
+        searchBarContainerView.addSubviews([searchTextField, searchBarButton])
         
+        headerView.snp.makeConstraints { make in
+            make.top.leading.trailing.equalTo(view.safeAreaLayoutGuide)
+            make.height.equalTo(52)
+        }
+        backButton.snp.makeConstraints { make in
+            make.centerY.equalToSuperview()
+            make.leading.equalToSuperview().offset(8)
+            make.width.height.equalTo(48)
+        }
+        searchTitleLabel.snp.makeConstraints { make in
+            make.centerX.centerY.equalToSuperview()
```

Comment:
centerX랑 centerY 같이할땐 center로 쓰면 더 좋을것같네요!!

```suggestion
make.center.equalToSuperview()
```

Reply:
오 꿀코드 감솨합니다